core/shared/src/test/scala/zio/prelude/CoherentSpec.scala

```diff
@@ -17,22 +18,38 @@ object CoherentSpec extends DefaultRunnableSpec {
       test("IdentityEqual") {
         val instance = implicitly[EqualIdentity[String]]
         assert(instance.identity)(isEmptyString) &&
-        assert(instance.combine("a", "b"))(equalTo("ab")) &&
-        assert(instance.equal("a", "a"))(isTrue)
+        assert("a".combine("b"))(equalTo("ab")) &&
+        assert(instance.equal("a", "a"))(isTrue) &&
+        assert("a".multiplyOption(5))(equalTo[Option[String]](Some("aaaaa"))) &&
+        assert("a".multiplyOption(-1))(equalTo[Option[String]](None)) &&
+        assert("a".multiplyOption(0))(equalTo[Option[String]](Some("")))
       },
       test("AssociativeEqual") {
         val instance = implicitly[AssociativeEqual[String]]
-        assert(instance.combine("a", "b"))(equalTo("ab")) &&
-        assert(instance.repeat("a")(5))(equalTo("aaaaa")) &&
-        assert(instance.repeat("a")(1))(equalTo("a")) &&
-        assert(instance.repeat("a")(0))(equalTo("a")) &&
-        assert(instance.equal("a", "a"))(isTrue)
+        assert("a".combine("b"))(equalTo("ab")) &&
+        assert("a".repeat(5))(equalTo("aaaaa")) &&
+        assert("a".repeat(1))(equalTo("a")) &&
+        assert("a".repeat(0))(equalTo("a")) &&
```

Comment:
This is the culprit! Java 11 added a new method on `String` also named `repeat` :smile: 
But if you change it to
```scala
        assert(instance.repeat("a")(5))(equalTo("aaaaa")) &&
        assert(instance.repeat("a")(1))(equalTo("a")) &&
        assert(instance.repeat("a")(0))(equalTo("a")) &&
```
it will work fine :+1: